lib/heuristics/concerns/scheduling_data_initialisation.rb

```diff
@@ -123,29 +123,24 @@ def collect_services_data(vrp)
       period = if service.visits_number == 1
                   nil
                 elsif has_only_one_day
-                  service[:minimum_lapse] ? (service[:minimum_lapse].to_f / 7).ceil * 7 : 7
+                  service.minimum_lapse ? (service.minimum_lapse.to_f / 7).ceil * 7 : 7
                 else
-                  service[:minimum_lapse] || 1
+                  service.minimum_lapse || 1
                 end
       @services_data[service.id] = {
+        vrp: service,
```

Comment:
the service is called through a symbol named vrp ? That's pretty confusing

Reply:
"I felt a great disturbance in the Force, as if millions of voices suddenly cried out in terror and were suddenly silenced. I fear something terrible has happened."
    ―Obi-Wan Kenobi

Reply:
This is provisory (you do not need to review this, I am still working on it but I am pushing to run some tests).

I am not satisfied with this name, I need to work on it but I want something short.
Because to call this we need to write `@services_data[id][:vrp]` (or any other symbol). I would like to avoid calling it service or original because it is too long ...